frontend/src/components/utils/Search.tsx

```diff
@@ -182,10 +183,18 @@ const Search: React.FC<Props> = React.memo(({ dashboard }) => {
   const { t } = useTranslation();
   const track = useAnalyticsTrack();
 
-  const [value, setValue] = useQueryParam("query");
+  const [queryParamQuery, setQueryParamQuery] = useQueryParam("query");
+  const [value, setValue] = React.useState<string>("");
   const [searchValue, setSearchValue] = React.useState<string | undefined>(
     undefined
   );
+
+  React.useEffect(() => {
```

Comment:
This should be replaces with initialValue as `queryParamQuery` (let's name it `queryValue`) as initial value of `value`'s `useState`.